### PR TITLE
Remove CSS vendor prefixes and switch to autoprefixer

### PR DESCRIPTION
**Pull request by dimaursu**
In some cases the prefixes were really old (border-radius, opacity).
Now you only have to write standard compliant CSS, and the prefixes for latest 2 versions will be added automatically.

### CODE REVIEW

**Comment by Flaburgan** [comment]
Hm, @include comes form scss. If this includes too old stuff, maybe we can do something about it there? Is it up to date ? Can't we configure what it includes?

**Comment by dimaursu** [comment]
@include is for bringing in mixin functionality. I deleted the ones dealing with opacity, border-shadow, and border-radius; The rest of them are useful for maintaining a consistent style, and to write shorter code. I think the ones left are reasonable.

File: Gemfile [hunk 1]
```diff
@@ -54,6 +54,7 @@ gem 'rack-cors', '0.2.9', :require => 'rack/cors'
 gem 'bootstrap-sass', '2.3.2.2'
 gem 'compass-rails',  '2.0.0'
 gem 'sass-rails',     '4.0.4'
+gem 'autoprefixer-rails'
```
Comment (jhass) [inline Gemfile]: Please add a `, '4.0.2.1'`

**Comment by dimaursu** [comment]
I'm not sure why Travis is failing. On my computer, the hovercard shows up just fine.

**Comment by jhass** [comment]
I'd care about these first: https://travis-ci.org/diaspora/diaspora/jobs/46186129#L429

**Comment by dimaursu** [comment]
Yey! Nailed it. I grepped back and forth this time, to make sure I don't miss something :)

**Comment by Flaburgan** [comment]
Knew it :P first change: the buttons don't have a gradient anymore:
capture du 2015-01-09 14 49 45

This also impacts the "sign in" button for example.

**Pull request merged**